application/src/main/java/com/github/pieter_groenendijk/model/DigitalProduct.java

```diff
@@ -0,0 +1,66 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "DigitalProduct")
+public class DigitalProduct extends Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long digitalProductId;
+
+    @Column(name = "title", nullable = false, length = 100)
```

Comment:
accessStatus != title?

Reply:
Helemaal overheengekeken, en renamed naar isAvailable. Maar ik pijns nog om het óf weg te halen óf te veranderen naar een Status 'comingSoon'. Zodat een beheerder het product wel kan toevoegen en adverteren maar nog niet daadwerkelijk in gebruik genomen kan worden.